server/util.go

```diff
@@ -108,3 +109,25 @@ func (t *slowLogTxn) Commit() (*clientv3.TxnResponse, error) {
 
 	return resp, errors.Trace(err)
 }
+
+// convertName converts variable name to a linux type name.
+// Like `AbcDef -> abc_def`.
+func convertName(str string) string {
+	strBytes := []byte(str)
```

Comment:
Seems no difference, if we use string, we have to convert travel item rune to []byte.